packages/utilities/src/memoize.ts

```diff
@@ -1,10 +1,15 @@
+import { Stylesheet } from '@uifabric/merge-styles';
+
+Stylesheet.getInstance().onReset(resetMemoizations);
```

Comment:
This seems like the wrong place to put this:

1. it adds a circular dep between styling and utilities.
2. it adds a side effect by just including this memoize function (which should really have nothing to do with styles in the first place)

Reply:
We discussed :) it doesn't (styling doesn't care about this.) utilities depends on mergeStyles.